libs/designsystem/modal/src/modal/services/modal.helper.ts

```diff
@@ -122,6 +122,16 @@ export class ModalHelper {
 
     await ionModal.present();
 
+    /**
+     * Due to somewhat unexpected behavior of the ion-modal not being focussed when keyboardClose
+     * is set to false we manually focus it, but only if focus is not already inside the modal
+     * See here: https://github.com/ionic-team/ionic-framework/issues/28775#issuecomment-1875475739
+     */
+    const focussedElementWithinModal = ionModal.contains(document.activeElement);
```

Comment:
Typo:
```suggestion
    const focusedElementWithinModal = ionModal.contains(document.activeElement);
```

> This word can take either double or single s, with the single option being highly preferred.

https://www.future-perfect.co.uk/grammar-tips/just-plain-wrong/is-it-focussed-or-focused